newm-server/src/main/kotlin/io/newm/server/features/cardano/repo/CardanoRepositoryImpl.kt

```diff
@@ -344,6 +347,32 @@ internal class CardanoRepositoryImpl(
         }
     }
 
+    override suspend fun getWalletNFTSongs(xpubKey: String, includeLegacy: Boolean): List<NFTSong> {
+        val queryWalletControlledLiveUtxosResponse = client.queryWalletControlledLiveUtxos(
```

Comment:
We do not need to query the LiveUtxos. You can use `queryWalletControlledUtxos()`. LiveUtxos is only if we're making transactions. The mobile app should not be making transactions. The LiveUtxos stuff adds extra processing we don't need.

Reply:
I don't see the  queryWalletControlledUtxos() call in the API.